src/Xamarin.Android.Build.Tasks/Tasks/ConvertCustomView.cs

```diff
@@ -65,7 +65,11 @@ public override bool Execute ()
 							update |= TryFixFragment (a, acw_map);
 						}
 						if (update) {
-							document.Save (file);
+							var lastModified = File.GetLastWriteTimeUtc (file);
+							if (document.SaveIfChanged (file)) {
+								Log.LogDebugMessage ($"Fixed up Custom Views in {file}");
+								MonoAndroidHelper.SetLastAccessAndWriteTimeUtc (file, lastModified, Log);
```

Comment:
I don't think we have to call `SetLastAccessAndWriteTimeUtc` here, because it gets called in `document.SaveIfChanged` -> `MonoAndroidHelper.CopyIfChanged`.